src/components/app.jsx

```diff
@@ -57,9 +60,9 @@ class App extends Component {
     return (
         <div>
           <ChatContainer
-            handleChangeAuthor={this.handleChangeAuthor.bind(this)}
-            handleChangeText={this.handleChangeText.bind(this)}
-            sendMessage={this.sendMessage.bind(this)}
+            handleChangeAuthor={this.handleChangeAuthor}
+            handleChangeText={this.handleChangeText}
+            sendMessage={this.sendMessage}
           />
```

Comment:
Try not to bind the function in the render function. because it will rebind every render. Consider instead binding it in the constructor.